lib/context.js

```diff
@@ -96,11 +96,20 @@ api.process = ({activeCtx, localCtx, options}) => {
           'Unsupported JSON-LD version: ' + ctx['@version'],
           'jsonld.UnsupportedVersion',
           {code: 'invalid @version value', context: ctx});
+      } else if(activeCtx.processingMode && activeCtx.processingMode.indexOf('json-ld-1.1') !== 0) {
```

Comment:
The above `if` throws an error, so our style is not to use an `else if` but to just use a regular `if` on a new line here, i.e. if an `if` exits the function via throwing or returning, there's no need to use `else`, just use `if`.